### ISSUE TITLE
Disk images updates for v24.0

### ISSUE THREAD

**Issue by ivanaamit** [gem5/gem5#1160]
- [x] X86 diskimage for 22.04 
- [x] x86 kernel extracted from the diskimage creation
- [x] x86 ubuntu 24.04
- [x] ARM diskimage for 22.04
- [x] ARM ubuntu 24.04
- [ ] ~~Arm kernel extracted from the diskimage creation~~ - cannot directly extract as the extract-vmliuz script doesnt work
- [x] RISCV diskiamge for 22.04 - Built, waiting on testing
- [ ] ~~RISCV kernel extracted from diskimage creation~~ - cannot directly extract as the extract-vmliuz script doesnt work
- [x] RISCV ubuntu 24.04 (it would be nice to have it) - disk image is made using preinstalled image, some issues with live install image
- [x] X86 npb image - address issue, otherwise we can make the benchmarks
- [x] Arm npb image
- [ ] riscv npb iamge

**Comment by BobbyRBruce** [comment]
@Harshil2107 : Is this done?

**Comment by Harshil2107** [comment]
yes
